Sources/Testing/Test.swift

```diff
@@ -210,17 +208,46 @@ extension Test {
     /// - ``Test/parameters``
     public var parameters: [Parameter]?
 
+    /// We can't store arbitrary traits on snapshots because they're not
+    /// generally encodable, but we can store comment traits.
+    public var comments: [Comment]
+
+    /// We can't store arbitrary traits on snapshots because they're not
+    /// generally encodable, but we can store tags.
+    public var tags: Set<Tag>
+
+    /// The set of bugs associated with this test.
+    ///
+    /// For information on how to associate a bug with a test, see the
+    /// documentation for ``Bug``.
+    public var associatedBugs: [Bug]
+
+    // This is not public because TimeValue is an internal type and we currently
```

Comment:
If we currently only need a property for the test suite, that means we're testing the test rather than the production code.

Reply:
Ah sorry, that comment is obsolete.